Makefile

```diff
@@ -18,8 +18,9 @@ CONROLLER_MANAGER_IMAGE_REPOSITORY := $(REGISTRY)/controller-manager
 SCHEDULER_IMAGE_REPOSITORY         := $(REGISTRY)/scheduler
 SEED_ADMISSION_IMAGE_REPOSITORY    := $(REGISTRY)/seed-admission-controller
 GARDENLET_IMAGE_REPOSITORY         := $(REGISTRY)/gardenlet
-IMAGE_TAG                          := $(shell cat VERSION)
-PUSH_LATEST_TAG                    := true
+PUSH_LATEST_TAG                    := false
+VERSION                            := $(shell cat VERSION)
+EFFECTIVE_VERSION                  := $(VERSION)-$(shell git rev-parse HEAD)
```

Comment:
I'm thinking about whether it makes sense checking for a dirty git state and assign it `$(VERSION)-$(shell git rev-parse HEAD)-dirty`, for example. We should avoid that someone accidentally overwrites an already pushed image that is at the moment being tested. WDYT?